docs/bridge.md

```diff
@@ -240,8 +240,63 @@ In the above configuration replace
 
 > Note: By default the openidconnect app will use the email of the user to match the user from the oidc userinfo endpoint with the ownCloud account. So make sure your users have a unique primary email.
 
+### Proxy
+
+The Proxy is responsible for:
+- Routing authentication request to ocis-konnectd which is the default bundled OIDC-Provider
+- Routing frontend requests to ocis-phoenix
+- Routing backend requests to owncloud 10
+
+#### Get it!
+```console
+$ git clone git@github.com:owncloud/ocis-proxy.git
+$ cd ocis-proxy
+$ make
+```
+
+### Configuration
+
+Edit ./ocis-proxy/config/.proxy-example.json
+
+```json5
+{
+	"routes": [
+		{
+			"endpoint": "/",
+			"backend": "http://localhost:9100" //Should point to phoenix
+		},
+		{
+			"endpoint": "/.well-known/openid-configuration",
+			"backend": "http://localhost:9130" // Should point to ocis-konnectd
+		},
+		{
+			"endpoint": "/konnect/",
+			"backend": "http://localhost:9130" // Should point to ocis-konnectd
+		},
+		{
+			"endpoint": "/signin/",
+			"backend": "http://localhost:9130" // Should point to ocis-konnectd
+		},
+		{
+			"endpoint": "/ocs/v1.php/",
+			"backend": "http://localhost:4444" // Your owncloud instance
+		},
+		{
+			"endpoint": "/remote.php/webdav/",
+			"backend": "http://localhost:4444"  // Your owncloud instance
```

Comment:
This will all need to be updated [to this](https://github.com/owncloud/ocis-proxy/pull/5/files#diff-9460e9f213c0c5dc0b398bf0d58f4f76R2-R61) once this is merged to reflect the new config structure: